tests/test_testPipeline.py

```diff
@@ -351,14 +379,14 @@ def testMockDiaPipelineTask(self):
         pipelineTests.assertValidOutput(task, result)
 
         self.butler.put(pandas.DataFrame(), "deepDiff_diaSrcTable", self.visitId)
-        self.butler.put(pandas.DataFrame(), "visitSsObjects", self.visitId)
+        self.butler.put(pandas.DataFrame(), "preloaded_ssObjects", self.groupId)
```

Comment:
It's a bit strange that you can drop `visitSsObjects`, because the default `DiaPipelineConnections` still takes them as inputs. That and the fix below means this code wasn't being exercised in the first place. 😞 

I'd recommend leaving this with `visitSsObjects`, since what we're supposed to be testing here is whether `MockDiaPipelineTask` is still representative of `DiaPipelineTask`, which has not been modified to support the new group-based dataset.

Reply:
tests/test_testPipeline.py::MockTaskTestSuite::testMockDiaPipelineTask fails when this is left with `visitSsObjects` but does not when this is left with `preloaded_SsObjects` (new capitalization).